### PR TITLE
Added support for direct usage of complex values in output block

### PR DESCRIPTION
**Pull request by rphh**

### CODE REVIEW

File: tests/integration/output_block_direct_values/resources/taste/work/actuator/SDL/src/actuator.pr [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            output result_values(right : 4);
+            /* CIF NEXTSTATE (827, 328), (70, 35) */
+            NEXTSTATE Wait;
+    endstate;
+    /* CIF state (587, 149), (70, 35) */
+    state Wait;
+        /* CIF input (569, 204), (105, 35) */
+        input give_me_data;
+            /* CIF output (521, 259), (200, 35) */
+            output result_data({first 2, second 3});
```
Comment (Lurkerpas) [inline tests/integration/output_block_direct_values/resources/taste/work/actuator/SDL/src/actuator.pr:30]: Just to make sure - can we have a test for a nested structure?

File: tests/integration/output_block_direct_values/resources/taste/output_block_direct_values.acn [hunk 2]
```diff
@@ -0,0 +1,12 @@
+OUTPUT-BLOCK-DIRECT-VALUES-DATAVIEW DEFINITIONS ::= BEGIN
+
+    --  In this ACN model, you can specify custom binary encoding rules for
```
Comment (Lurkerpas) [inline tests/integration/output_block_direct_values/resources/taste/output_block_direct_values.acn:3]: Remove the commented out code

File: tests/integration/output_block_direct_values/resources/taste/work/actuator/SDL/src/actuator.pr (hunk 1 above)
Comment (rphh) [reply tests/integration/output_block_direct_values/resources/taste/work/actuator/SDL/src/actuator.pr]: Added, waiting for CI (it worked on my machine)

File: tests/integration/output_block_direct_values/resources/taste/output_block_direct_values.acn (hunk 2 above)
Comment (rphh) [reply tests/integration/output_block_direct_values/resources/taste/output_block_direct_values.acn]: removed

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**